test/jdk/jdk/incubator/vector/templates/Unit-header.template

```diff
@@ -1184,8 +1184,15 @@ public class $vectorteststype$ extends AbstractVectorTest {
     static final List<BiFunction<Integer,Integer,$type$[]>> $TYPE$_SHUFFLE_GENERATORS = List.of(
             withToStringBi("shuffle[random]", (Integer l, Integer m) -> {
                 $type$[] a = new $type$[l];
+#if[ByteMax]
```

Comment:
You could probably simplify this a bit by rename argument `m` to `upper` and updating it for `ByteMax`, thereby the loop code remains the same.

Reply:
Hi @PaulSandoz thanks for looking at this PR.  I'v updated the patch to make sure the loop code remain the same. Could you please to take a look at it again? Thanks!